cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -53,37 +48,6 @@ def _install():
         yum_install(source)
 
 
-def _deploy_resources():
-    deploy(
-        src=join(CONFIG_PATH, 'rabbitmq_ulimit.conf'),
-        dst=FD_LIMIT_PATH
-    )
-    deploy(
-        src=join(CONFIG_PATH, 'rabbitmq-definitions.json'),
-        dst=join(HOME_DIR, 'definitions.json'),
-        render=False
-    )
-    deploy(
-        src=join(CONFIG_PATH, 'rabbitmq-env.conf'),
-        dst=join(HOME_DIR, 'rabbitmq-env.conf')
-    )
-    chown(RABBITMQ, RABBITMQ, HOME_DIR)
-
-
-def _enable_plugins():
```

Comment:
nice!

Reply:
is it also possible to specify users in the definitions file or similar? then we would also be able to get rid of `_delete_guest_user` and `_create_rabbitmq_user`

Reply:
Probably not (or, best not to even if we can, because that would entail a plaintext password)